keras_cv/layers/object_detection/roi_align.py

```diff
@@ -192,10 +194,14 @@ def multilevel_crop_and_resize(
     """
 
     with tf.name_scope("multilevel_crop_and_resize"):
-        levels = list(features.keys())
-        min_level = int(min(levels))
-        max_level = int(max(levels))
-        features_shape = tf.shape(features[min_level])
+        levels_str = list(features.keys())
+        # Levels are represented by strings with a prefix "P" to represent
+        # pyramid levels. The integer level can be obtained by looking at
+        # the value that follows the "P".
+        levels = [int(level_str[1:]) for level_str in levels_str]
```

Comment:
Is this strictly required? If we take the max using the strings, shouldn't we get the same result due to the String sort order matching the int order in this case?

Reply:
You are right, if you use max using strings you would get the same result and that's what I initially did. However, it seemed easier to convert it to an int list upfront to avoid messing with the operations on Lines 219, 260, 263 & 280 (the line numbers are the numbers after the edits).